app/models/ability.rb

```diff
@@ -51,13 +52,14 @@ def initialize(user)
       can [:view_and_add_notes, :show], ApplicationLetter
       can [:print_applications], Event
       can :manage, Request
+      cannot :apply, Event
       cannot :check, ApplicationLetter
     end
     if user.role? :organizer
       can [:index, :show], Profile
       can [:index, :show, :view_and_add_notes, :update_status], ApplicationLetter
       cannot :update, ApplicationLetter
-      can [:view_applicants, :edit_applicants, :view_participants, :print_applications, :manage, :view_material, :upload_material, :print_agreement_letters, :download_material, :view_unpublished], Event
+      can [:view_applicants, :edit_applicants, :view_participants, :print_applications, :view_material, :upload_material, :print_agreement_letters, :download_material, :view_unpublished, :new, :edit, :destroy], Event
```

Comment:
You still need to add an extra permission like :view_custom_application_fields, and update the ability spec accordingly.
Further, I found two usages of :manage, Event. I guess those are the one that cause your tests to fail atm.
Use your editor or execute something like grep "can? :manage, Event -r app" to find them